vic/vic_run/src/lakes.eb.c

```diff
@@ -1901,13 +1900,13 @@ water_balance(lake_var_struct *lake,
     volume_save = lake->volume;
     ErrorFlag = get_depth(lake_con, lake->volume - lake->ice_water_eq, &ldepth);
     if (ErrorFlag == ERROR) {
-        log_err("Something went wrong in get_depth; record = %d, "
-                "volume = %f, depth = %e", rec, lake->volume, ldepth);
+        log_err("Something went wrong in get_depth; "
+                "volume = %f, depth = %e", lake->volume, ldepth);
```

Comment:
So do we know lose any info on when things went wrong? Same for other `log_err` occurrences.